FiveOhFirstDataCore.Core/Services/RosterService.cs

```diff
@@ -87,7 +87,7 @@ public async Task<(HashSet<int>, HashSet<string>)> GetInUseUserDataAsync()
             {
                 ids.Add(x.Id);
 
-                if (x.Slot < Data.Slot.InactiveReserve)
```

Comment:
Inactive reserves are not taking into account when getting nickname data. A new trooper can pick a nickname that someone in inactive reserves currently has.

Reply:
Confirmed with Recuiter InactiveReserves are protected